### PR TITLE
[wip] add spark

### PR DESCRIPTION
**Pull request by slopp**
This PR adds an asset that uses spark 

Right now the asset uses a local pyspark session on local and databricks for branch/prod

Materializing the databricks asset can take a long time (it creates a new databricks cluster scoped to the job), but this should let us spin down the existing demo databricks cluster, which is running ~$500 /month 😱 

To review, I'd recommend looking at the most recent run in the branch deployment which materialized the `big_orders` asset. 

There are a few spurious changes (readme updates, setting defaults for asset runtime config, fixing tests)... had to do something while the databricks clusters failed.

### CODE REVIEW

File: hooli_data_eng/repository.py [hunk 1]
```diff
@@ -191,6 +200,15 @@ def get_env():
     selection=AssetSelection.keys(["forecasting","predicted_orders"]),
 )
 
+databricks_job = define_asset_job("databricks_job", 
+    selection=AssetSelection.keys(["forecasting","big_orders"]),
+    config = {
+        "resources": {
```
Comment (slopp) [inline hooli_data_eng/repository.py:206]: Doesn't work

**Comment by slopp** [comment]
Updated the databricks token in the user-cloud secret:

```
echo -n "<Databricks User Token" | base64
kubectl -n data-eng-prod edit secret hooli  # using the hooli-user-cluster kubernetes context 
```

Also bumped the databricks secrets `snowflake_user`, `snowflake_password`, and `snowflake_account` using:

```
pip install databricks-cli
databricks configure --token
databricks secrets put --scope dagster-test --key snowflake_account
databricks secrets put --scope dagster-test --key snowflake_account
```

**Review by benpankow — APPROVED** [review]
Looks great to me!

File: dbt_project/README.md [hunk 2]
```diff
@@ -1,15 +1,7 @@
-Welcome to your new dbt project!
+This dbt project integrates with Dagster in a few ways:
```
Comment (benpankow) [inline dbt_project/README.md:1]: thanks for changing this haha

**Pull request merged**